app/code/Magento/Wishlist/Api/data/WishlistInterface.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+namespace Magento\Wishlist\Api\data;
+
+interface WishlistInterface
+{
+    /**#@+
+     * Constants defined for keys of  data array
+     */
+    const NAME = 'name';
```

Comment:
Wishlist name is immutable in the current implementation, why we need this field in api?

Reply:
Should api not have method to retrieve name? Or should I only remove NAME constant and keep getName method and remove setName method? If so, why remove constant?